google/users.go

```diff
@@ -182,8 +182,8 @@ func isMatchingType(i interface{}, findType string) map[string]interface{} {
 	return nil
 }
 
-// setStringFromInterface gets a string from an interface{}, and assigns it to a map
-func setStringFromInterface(i interface{}, m map[string]string, key string) {
+// setStringFromInterface gets a string from an any, and assigns it to a map
```

Comment:
You don't need to change this, but I wonder what a better term for "an any" would be -- "a generic"?

Reply:
Or just a `value`.

Reply:
Huh, I don't know. I'll come up with something because it doesn't make sense at the moment.

Reply:
The technical term is "empty interface". I'll use that.